src/styles/RecipeFinder.css

```diff
@@ -0,0 +1,82 @@
+.dish-input {
+  padding: 0.2rem;
+  border: 0;
+  font-size: 1.3rem;
+  &:focus {
+    border: 0
+  }
+}
+
+.dish-input:focus-visible {
+  outline: none;
```

Comment:
Why remove the focus outline?

Reply:
Added outline on the input's container instead